File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-06/plagiarized/L4/09/Level4.kt
package IR

/**
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level4 {
    @JvmStatic
    fun main(args: Array<String>) {
        proses()
    }

    @JvmStatic
    fun proses() {
        val `in` = java.util.Scanner(System.`in`)
        val angka = IntArray(10)
        for (a in 0 until 10) {
            print("Read a number: ")
            angka[a] = `in`.nextInt()
        }

        for (b in 9 downTo 0) {
            println(angka[b])
        }
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L4/09/Level4.py
import sys

def proses():
    angka = [0] * 10
    for a in range(10):
        print("Read a number: ", end="")
        angka[a] = int(input())

    for b in range(9, -1, -1):
        print(angka[b])

if __name__ == "__main__":
    proses()

